README.md

```diff
@@ -0,0 +1,133 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b> hello-react-front-end </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] [OPTIONAL] I can see that this sectional has been removed from the body of the README so kindly remove it from the Table of content as well.